composer.json

```diff
@@ -7,7 +7,23 @@
   "repositories": [
     {
       "type": "composer",
-      "url": "https://wpackagist.org"
+      "url":"https://wpackagist.org"
+    },
+    {
+      "type": "package",
+      "package": {
+        "name": "gravityforms/gravityforms",
+        "version": "2.4.21.3",
+        "type": "wordpress-plugin",
+        "dist": {
+          "type": "zip",
+          "url": "https://www.gravityhelp.com/wp-content/plugins/gravitymanager/api.php?op=get_plugin&slug=gravityforms&key={%GRAVITYFORMS_KEY}"
```

Comment:
Does that mean that only people with GF key will be able to work on the plugin?

Reply:
@kasparsd Yea, it is. :thinking: This might need some further tooling.

Reply:
I think we should skip GravityForms unless the develop installs it as a `dev` specifically with composer. We can probably move the whole GravityForms install process into the Travis-CI script.

Reply:
@kasparsd :point_up: What do you think?

Reply:
Yes, that is a good idea! We could even have another composer.json file for installing these external dependencies.